packages/desktop/src/main/main.ts

```diff
@@ -375,22 +375,31 @@ app.on('ready', async () => {
     `${SOCKET_IO_SECRET}`,
   ]
 
+  logger.info('Fork argvs for backend process', forkArgvs)
+
   const backendBundlePath = path.normalize(require.resolve('backend-bundle'))
   try {
     closeHangingBackendProcess(path.normalize(path.join('backend-bundle', 'bundle.cjs')), path.normalize(appDataPath))
   } catch (e) {
     logger.error('Error occurred while trying to close hanging backend process', e)
   }
 
-  backendProcess = fork(backendBundlePath, forkArgvs)
+  backendProcess = fork(backendBundlePath, forkArgvs, {
+    env: {
+      NODE_OPTIONS: '--experimental-global-customevent',
+      DEBUG: 'backend*,quiet*,state-manager*,desktop*,utils*,identity*,common*,libp2p:connection-manager:auto-dial',
```

Comment:
The debug flag doesn't seem to be inherited from desktop anymore for some reason.